buildtest/settings/config.yml

```diff
@@ -1,45 +1,23 @@
 system:
   generic:
-    # specify list of hostnames where buildtest can run for given system record
-    hostnames: [".*"]
-
-    # system description
```

Comment:
please dont commit `buildtest/settings/config.yml` this was a result of running the regression test which autoupdates the file. Don't do `git add .` or something. Just copy the content from `devel` branch and paste it in that should avoid showing this commit diff for this file

Reply:
noted

Reply:
@kingxleyy i still see this file ``buildtest/settings/config.yml` in commit diff